src/test/java/org/opentripplanner/routing/algorithm/mapping/RaptorPathToItineraryMapperTest.java

```diff
@@ -82,8 +106,41 @@ public void createItineraryTestZeroDurationEgress(int LAST_LEG_COST) {
     );
   }
 
+  /**
+   * Create a minimalist path FlexAccess-->Transfer-->Egress (without transit) and check that the 3 legs are properly mapped in the itinerary.
+   *
```

Comment:
```suggestion
```

Reply:
The line is to long, prettier does NOT format JavaDoc, so you must do it youself.